src/app/api/trpc/[trpc]/route.tsx

```diff
@@ -19,7 +16,7 @@ const createContext = async ({
 }
 
 const handler = (request: Request) => {
-  trpcLogger.info(`incoming request ${request.url}`)
+  console.info(`incoming request ${request.url}`)
```

Comment:
Should this go to the new logger instead of console?

Reply:
The new logger doesn't work in any front end components so it has to be console instead.